### ISSUE TITLE
Pull Request information

### ISSUE THREAD

**Issue by eddiejaoude** [TransformCore/documentation-templates#23]
Some good advice on here, is it work including the important parts & then referencing the rest? https://github.com/blog/1943-how-to-write-the-perfect-pull-request

**Comment by rorywoodward** [comment]
I like it (apart from the overuse of emoji in the "Use emoji to clarify tone" comment). I think you are right - it can be summarised into key points then referenced.

**Comment by eddiejaoude** [comment]
Yes, a lot of emoji over use (I would usually put a smily face here)
